charts/grafana/values.yaml

```diff
@@ -9,7 +9,7 @@ tolerations: []
 images:
   grafana:
     repository: quay.io/astronomer/ap-grafana
-    tag: 8.3.3-1
+    tag: 8.3.7
```

Comment:
Do we want to use 8.3.7 or 8.4.5?

Reply:
8.4.5 is very latest and both version didn't fix go package vulnerabilities